src/mono/wasm/runtime/managed-exports.ts

```diff
@@ -38,6 +23,23 @@ export function init_managed_exports(): void {
     if (!runtimeHelpers.runtime_interop_exports_class)
         throw "Can't find " + runtimeHelpers.runtime_interop_namespace + "." + runtimeHelpers.runtime_interop_exports_classname + " class";
 
+    const install_sync_context = mono_method_resolve(
```

Comment:
Would this managed code exist in the non-MT build ? 
We have `import MonoWasmThreads from "consts:monoWasmThreads"`
and `if (MonoWasmThreads )  {`

And perhaps some `#if` ON c# side.

Reply:
The current behavior is that we have the default synchronization context, which sends everything to the threadpool. We want to always install this even if threads are inactive, so that users get consistent behavior.